Description: replace ava by tape
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2022-10-09

--- a/packages/buble/test/test.js
+++ b/packages/buble/test/test.js
@@ -1,7 +1,7 @@
-const test = require('ava');
+const test = require('tape');
 const rollup = require('rollup');
 
-const buble = require('..');
+const buble = require('@rollup/plugin-buble');
 
 process.chdir(__dirname);
 
@@ -30,4 +30,5 @@
   const generated = await getChunksFromBundle(bundle);
   t.is(generated.length, 1);
   t.is(generated[0].code, 'function main () { return 42; }\n\nexport { main as default };\n');
+  t.end();
 });
